modules/aruco/include/opencv2/aruco.hpp

```diff
@@ -303,6 +303,8 @@ class CV_EXPORTS_W Board {
     /// vector of the identifiers of the markers in the board (same size than objPoints)
     /// The identifiers refers to the board dictionary
     CV_PROP_RW std::vector< int > ids;
+
+    CV_PROP Point3f rightBottomBorder;
```

Comment:
Please add documentation line for the property.

Reply:
added

Reply:
![image](https://user-images.githubusercontent.com/22337800/154146907-133bbb0a-78eb-4eb4-9d0c-f26d4625bad4.png)